components/VerifiableCredentialResponse.yml

```diff
@@ -12,9 +12,9 @@ info:
 paths:
 components:
   schemas:
-    IssueCredentialSuccess:
+    VerifiableCredentialResponse:
```

Comment:
Hard to tell from the diff if this covers the `GET /credentials/{id}` case, but if it does, then this should resolve the issue.

Reply:
We confirmed during the call that this PR does apply the intended change.